op-deployer/pkg/deployer/standard/standard-versions-mainnet.toml

```diff
@@ -3,6 +3,26 @@
 #   * proxied             : specify a standard "implementation_address"
 #   * neither             : specify neither a standard "address" nor "implementation_address"
 
+# Holocene https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.8.0
+["op-contracts/v1.8.0"]
```

Comment:
Not sure if we want to skip this in favor of the already existing - `v1.8.0-rc.4`